docs/user/security.rst

```diff
@@ -2,8 +2,8 @@
 ..    https://readthedocs.org/.well-known/security.txt
 ..    https://readthedocs.org/security/
 
-Security
-========
+Security practice
+=================
```

Comment:
I believe this is actually a page on reporting security issues, I think. So something like:

```suggestion
Security reports
================
```

I think the `Account security` stuff is probably better said in the `Security policy` doc, so we should probably just add a `seealso` there, and remove that section.

Reply:
Good spot! I'll move it! I think it sounds fine to have "Security reports" in this section still -- it's still a procedure implementing the Security Policy and I think 99% of people that want to report security issues will not look in the How-to section of a documentation but rather for a security policy.